TA-puppet-alert-actions/bin/ta_puppet_alert_actions/pie/bolt.py

```
@@ -25,6 +25,19 @@ def post(endpoint, req, headers):
 
   return json.loads(r.text)['job']
 
+def post_plan(endpoint, req, headers):
+
+  try:
+    r = requests.post(endpoint, json=req, headers=headers, verify=False)
+  except:
+    print('Unexpected error:', sys.exc_info()[0])
+    raise
```

You might want to bubble up an exception to the caller?